### PR TITLE
Created a Mindee glossary

### PR DESCRIPTION
**Pull request by fakela**
Create mindee glossary.md #200 

Would need more ideas and definitions of terminologies used in Mindee

### CODE REVIEW

**Comment by fharper** [comment]
It's fine if we start small, we'll build the glossary while revising the documentation

**Comment by fharper** [comment]
@fakela: we can add something about: if you think a term should be explained, please let us know on Slack. And maybe something about the fact that this page is a work in progress, so it doesn't look too empty, but we could merge it now so it is already profitable for our actual users, WDYT?

**Comment by fakela** [comment]
> @fakela: we can add something about: if you think a term should be explained, please let us know on Slack. And maybe something about the fact that this page is a work in progress, so it doesn't look too empty, but we could merge it now so it is already profitable for our actual users, WDYT?

Yes, we can. I will share this tomorrow on slack too

**Review by fharper — COMMENTED** [review]
Should we add something about the fact that it's a work in progress, or that we will continually add terms in it? Since it's a bit empty for now.

File: get-started/mindee-glossary.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+next:
+  pages:
+    - build-your-first-document-parsing-api
+    - invoice-ocr
+  description: 'mindee glossary'
+---
+## Mindee glossary
+This glossary introduces common Mindee terminology. You will frequently encounter these concepts throughout this docs as Mindee is centered around them
+
+- **Models**: A model is a machine learning file that can be trained to carry out specific tasks such as extracting the total amount info from receipts etc.
```
Comment (fharper) [inline get-started/mindee-glossary.md:13]: Alphabetical order :)

**Review by fharper — CHANGES_REQUESTED** [review]

**Review by fharper — APPROVED** [review]
LGTM

**Pull request merged**